default.cfg

```diff
@@ -1,41 +1,54 @@
-alwaysrun		false
-chatmacro0		"No"
-chatmacro1		"I'm ready to kick butt!"
-chatmacro2		"I'm OK."
-chatmacro3		"I'm not looking too good!"
-chatmacro4		"Help!"
-chatmacro5		"You suck!"
-chatmacro6		"Next time, scumbag..."
-chatmacro7		"Come here!"
-chatmacro8		"I'll take care of it."
-chatmacro9		"Yes"
-color_depth		"hicolor"
-detaillevel		0
-fullscreen		false
-joyb_fire		0
-joyb_speed		2
-joyb_strafe		1
-joyb_use		3
-key_down		115
-key_fire		157
-key_left		172
-key_right		174
-key_speed		182
-key_strafe		184
-key_strafeleft		97
-key_straferight		100
-key_up		119
-key_use		32
-mb_used		2
-mouse_sensitivity		5
-mouseb_fire		0
-mouseb_forward		1
-mouseb_strafe		2
-music_volume		8
-screenblocks		10
-sfx_volume		8
-show_messages		true
-snd_channels		32
-use_joystick		false
-use_mouse		true
-usegamma		0
+alwaysrun		true
+automap_plotter_style		Deep
+chatmacro0		"No"
+chatmacro1		"I'mreadytokickbutt!"
+chatmacro2		"I'mOK."
+chatmacro3		"I'mnotlookingtoogood!"
+chatmacro4		"Help!"
+chatmacro5		"Yousuck!"
+chatmacro6		"Nexttime,scumbag..."
+chatmacro7		"Comehere!"
+chatmacro8		"I'lltakecareofit."

```

Comment:
Config manager is now fixed and will not make those again.
Interestingly, it seems it was buggy from the start